src/signals/incident/definitions/wizard-step-2-vulaan/verlichting-icons.ts

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2021 Gemeente Amsterdam
+
+export const gevel_armatuur = `<svg width="40px" height="40px" viewBox="0 0 40 40" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+<title>icon-gevelarmatuur</title>
+<g id="icoon/Icon_40_GevelArmatuur" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+  <circle id="Oval" stroke="#000000" stroke-width="2" fill="#FFFFFF" cx="20" cy="20" r="19"></circle>
+  <ellipse id="Oval" stroke="#000000" stroke-width="2" fill="#FEC813" transform="translate(20.000000, 20.000000) rotate(-90.000000) translate(-20.000000, -20.000000) " cx="20" cy="20" rx="11" ry="5"></ellipse>
+  <rect id="Rectangle" fill="#000000" transform="translate(12.000000, 20.000000) rotate(-90.000000) translate(-12.000000, -20.000000) " x="10" y="17" width="4" height="6"></rect>
+  <polygon id="Rectangle" fill="#000000" points="4.73440982 8.1819567 9.90643311 4.46923828 9.90643311 36.4692383 4.73440982 31.8146083 1.90643311 25.3724969 1.90643311 15.3032558"></polygon>
+</g>
+</svg>
+`
```

Comment:
All SVG declarations in this file can (and should) be separate SVG file. They can be imported by using a resource query, like so:
```javascript
import gevelArmatuurIcon from './gevel-armatuur.svg?url'
``
This will ensure that any reference to `gevelArmatuurIcon` will be used as a URL instead of a component.

Reply:
I will do that in the klok PR, okay?